src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -2139,5 +2144,31 @@ .maxstack  4
 }");
             });
         }
+
+        [Fact]
+        public void DuplicateDeclaration()
```

Comment:
>DuplicateDeclaration [](start = 20, length = 20)

Is it possible to add similar test for VB?

Reply:
Good question!
It looks that:
1. It does not allow to execute
'Dim x = 3'
in the immediate window even for non declared variable.
It returns:
Statements of type 'FieldDeclaration' are not allowed in the Immediate window.

2. If one tries to execute 'x = 3' in the immediate window, it would either use an existing 'x' from code or from previous statements executed, or introduce a new variable.

Therefore, it seems that there is no such problem in VB. So, I do not see a suitable test for VB.

---
In reply to: [161291906](https://github.com/dotnet/roslyn/pull/24198#discussion_r161291906) [](ancestors = 161291906)